sdk/src/main/java/io/radar/sdk/RadarSettings.kt

```diff
@@ -118,8 +120,27 @@ internal object RadarSettings {
         getSharedPreferences(context).edit { putBoolean(KEY_TRACKING, tracking) }
     }
 
+    internal fun getListenToServerTrackingOptions(context: Context): Boolean {
+        return getSharedPreferences(context).getBoolean(KEY_SERVER_TRACKING, false)
+    }
+
+    internal fun setListenToServerTrackingOptions(context: Context, tracking: Boolean) {
+        getSharedPreferences(context).edit { putBoolean(KEY_SERVER_TRACKING, tracking) }
+    }
+
+    internal fun revertToFallbackTrackingOptions(context: Context) {
+        val fallbackTrackingOptions = getFallbackTrackingOptions(context)
+        if (fallbackTrackingOptions != null) {
+            setTrackingOptions(context, fallbackTrackingOptions)
+        }
+    }
+
     internal fun getTrackingOptions(context: Context): RadarTrackingOptions {
-        val optionsJson = getSharedPreferences(context).getString(KEY_TRACKING_OPTIONS, null)
+        return getTrackingOptionsByKey(context, KEY_TRACKING_OPTIONS)
+    }
+
+    private fun getTrackingOptionsByKey(context: Context, key: String): RadarTrackingOptions {
+        val optionsJson = getSharedPreferences(context).getString(key, null)
```

Comment:
Would it make sense to prefix these names with "TRACKING_OPTIONS" or something to keep it unique to this feature?

Reply:
hmm this accept a `key` as a string. currently these are the keys - I was trying to maintain stability with existing keys:

```
    private const val KEY_TRACKING_OPTIONS = "tracking_options"
    private const val KEY_FALLBACK_TRACKING_OPTIONS = "fallback_tracking_options"
```

I wanted to make sure upgrades between SDKs also maintained the history, so tried not to change the original tracking option key name